build.sbt

```diff
@@ -5,7 +5,7 @@ val Scala2_13 = "2.13.10"
 val Scala3    = "3.2.2"
 
 val isScala3 = Def.setting(
-  CrossVersion.partialVersion(scalaVersion.value).exists(_._1 == 3)
+  CrossVersion.partialVersion(scalaVersion.value).exists { case (epic, _) => epic == 3 }
```

Comment:
"epic" is nice! :)
I think you mean "epoch", right?
(Let's leave _epic_ though, I like it... :)

Reply:
I thought the technical term for Scala3 style versioning is epic.major.minor but that might just be my brain playing up. I can change it to epoch if you want.

Reply:
No, "epic" is better... Thank you!

Reply:
we can use `scalaBinaryVersion.value == “3”`

Reply:
That is much nicer, done